automaxlair/max_lair_instance.py

```diff
@@ -182,3 +184,89 @@ def __init__(
         self.row = row
         self.col = col
         self.downstream_nodes = downstream_nodes
+
+
+class Weather(object):
```

Comment:
Do you think it would make sense to have a `Field` object that had `weather` and `terrain` attributes that are just strings? You could check if the weather is raining with `field.weather == 'rain'` for example.

Later down the road, it would be possible to add two 'sides' lists that can have effects such as Reflect etc.

Reply:
yes totally ! I wasn't sure on how to do it. Because there wasn't any enum, I tried to replicate it to the closest but string works fine too. Also it might be better for debugging purpose (having rain, instead of  is probably easier)